test/ctia/test_helpers/atom.clj

```diff
@@ -0,0 +1,83 @@
+(ns ctia.test-helpers.atom
+  (:require [ctia.store :refer [stores]]
+            [ctia.test-helpers.core :refer [with-properties]]))
+
+(defn fixture-properties:atom-memory-store [f]
+  ;; Set properties to enable the atom store
+  (with-properties ["ctia.store.actor" "atom"
+                    "ctia.store.atom.actor.mode" "memory"
+                    "ctia.store.campaign" "atom"
+                    "ctia.store.atom.campaign.mode" "memory"
+                    "ctia.store.coa" "atom"
+                    "ctia.store.atom.coa.mode" "memory"
+                    "ctia.store.exploit-target" "atom"
+                    "ctia.store.atom.exploit-target.mode" "memory"
+                    "ctia.store.feedback" "atom"
+                    "ctia.store.atom.feedback.mode" "memory"
+                    "ctia.store.identity" "atom"
+                    "ctia.store.atom.identity.mode" "memory"
+                    "ctia.store.incident" "atom"
+                    "ctia.store.atom.incident.mode" "memory"
+                    "ctia.store.indicator" "atom"
+                    "ctia.store.atom.indicator.mode" "memory"
+                    "ctia.store.judgement" "atom"
+                    "ctia.store.atom.judgement.mode" "memory"
+                    "ctia.store.verdict" "atom"
+                    "ctia.store.atom.verdict.mode" "memory"
+                    "ctia.store.sighting" "atom"
+                    "ctia.store.atom.sighting.mode" "memory"
+                    "ctia.store.ttp" "atom"
+                    "ctia.store.atom.ttp.mode" "memory"]
+    (f)))
+
+(defn fixture-properties:atom-durable-store [f]
+  ;; Set properties to enable the atom store
+  (with-properties ["ctia.store.actor" "atom"
+                    "ctia.store.atom.actor.mode" "durable"
+                    "ctia.store.atom.actor.path" "/tmp/durable-actor.edn"
```

Comment:
I think we might want to have all those files on a shared top folder like for example /tmp/ctia/<file>.edn so we can easily delete everything.